applications/luci-app-vpnbypass/root/usr/libexec/rpcd/luci.vpnbypass

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+# Copyright 2021 Stan Grishin (stangri@melmac.net)
+# shellcheck disable=SC1091,SC2039
+
+# TechRef: https://openwrt.org/docs/techref/rpcd
+
+. /lib/functions.sh
+. /usr/share/libubox/jshn.sh
+
+pkgName="vpnbypass"
+
+is_enabled() { uci -q get "${1}.config.enabled"; }
+is_running() { iptables -t mangle -L | grep -q VPNBYPASS && echo '1' || echo '0'; }
+get_version() { grep -A2 -w "Package: $1$" /usr/lib/opkg/status | sed -n 's/Version: //p'; }
+print_json_bool() { json_init; json_add_boolean "$1" "$2"; json_dump; json_cleanup; }
+logger() { /usr/bin/logger -t "$pkgName" "$@"; }
+
+get_init_list() {
+	local name="$1"
+	json_init
+	json_add_object "$name"
+	json_add_boolean 'enabled' "$(is_enabled "$name")"
+	json_add_boolean 'running' "$(is_running "$name")"
+	json_close_object
+	json_dump
+	json_cleanup
+}
+
+set_init_action() {
+	local name="$1" action="$2" cmd
```

Comment:
The `$name` variable here allows for arbitrary shell command injection. Please validate it through something like `[ -f "/etc/init.d/$name" ]` and return something like `{ "error": "Init script not found" }` if the condition is not satisfied.